src/components/HeroSection.tsx

```diff
@@ -0,0 +1,46 @@
+          fill="#ffffff"
+        />
+      </svg>
+      <H2 className="text-white">無聊想找伴或參加活動嗎</H2>
+      <h1 className="text-[4rem]/[4.8rem] font-thin -tracking-[0.012em] text-white xl:text-center xl:text-[7.5rem]/[9rem]">
```

Comment:
設計稿字體 `font-weight: 250`，最接近的應該是 `font-extralight (font-weight: 200;)`
```suggestion
      <h1 className="text-[4rem]/[4.8rem] font-extralight -tracking-[0.012em] text-white xl:text-center xl:text-[7.5rem]/[9rem]">
```